frontend/src/metabase/qb/components/modes/NativeMode.jsx

```diff
@@ -1,10 +1,11 @@
 /* @flow */
 
 import type { QueryMode } from "metabase/meta/types/Visualization";
+import CompoundQueryAction from "../actions/CompoundQueryAction";
 
 const NativeMode: QueryMode = {
     name: "native",
-    actions: [],
+    actions: [CompoundQueryAction],
```

Comment:
Is this intentionally only available to native queries although `should suggest a compound query for a mbql query` test case implies that it should be available for MBQL queries as well?